### ISSUE TITLE
Mark a datastore as deprecated

### ISSUE THREAD

**Issue by modir** [OpenNebula/one#4418]
**Description**
It would be great if I could mark a datastore as "deprecated" (or similar word). This would then not allow anymore to create new VMs on this datastore.

**Use case**
We have here one datastore which we want to fade out. I always tell my colleagues to not put any new VMs on this datastore. But some of them are either very lazy or stupid and they create new VMs on this datastore.

**Interface Changes**
At the moment datastore can only be in the states "enabled" or "disabled". Probably the easiest would be to add a third state "deprecated" which would then enforce that no new VMs can be created on this datastore.

**Additional Context**
Add any other context or screenshots about the feature request here. Or any other alternative you have considered to addressed this new feature.

## Progress Status
- [ ] Branch created
- [ ] Code committed to development branch
- [ ] Testing - QA
- [ ] Documentation
- [ ] Release notes - resolved issues, compatibility, known issues
- [ ] Code committed to upstream release/hotfix branches
- [ ] Documentation committed to upstream release/hotfix branches

**Comment by kvaps** [comment]
Hey @modir, you can just remove ACL which allows to use this datastore, they wil not even see it afterwards

**Comment by modir** [comment]
Hi @kvaps , Sadly no. They use the oneadmin Account all the time so they see everything. (We are only a small team so no need for many different accounts in general.)

**Comment by modir** [comment]
Please keep this issue open.

**Comment by rsmontero** [comment]
@modir 

onedatastore disable wouldn't do the job?

```
disable
        Disables the given Datastore. Only available for System Datastores
```

**Comment by modir** [comment]
@rsmontero Will then the already existings VMs which have disks on this datastore still run? The way I expect "disable" to behave: no. Hence I need something for "still allow VMs to run from this datastore but do not allow to create something new".

**Comment by rsmontero** [comment]
It actually works as you describe. When a new VM has to be scheduled the scheduler filters out any DISABLED datastore. VMs running on the Datastore will not be affected.

**Comment by rsmontero** [comment]
This is spelled out in the documentation

https://docs.opennebula.io/6.0/management_and_operations/storage_management/datastores.html#disable-a-system-datastore

**Comment by modir** [comment]
OK. In this case the wording is from my point of view missleading.

Is it the same for image stores?

**Comment by rsmontero** [comment]
Not currently not implemented. We can reword this issue and use DISABLE in Image datastores to prevent the creation of new images, what do you think?

**Comment by modir** [comment]
Yes, good idea. Will you do the rewording so it fits your needs? Or should I do it?

And what I always wanted to have is that the GUI has "help" icons which link to the relevant article the documentation. This would have helped me in this situation to look it up at the right place. But for this I can open up another issue.

**Comment by rsmontero** [comment]
I'll take thanks for the feedback!
